src/test/java/uk/gov/hmcts/reform/sscs/jms/listener/TribunalsHearingsEventTopicListenerTest.java

```diff
@@ -52,16 +52,19 @@ class TribunalsHearingsEventTopicListenerTest {
     @Mock
     private IdamService idamService;
 
+    @Mock
+    private SscsCaseData sscsCaseData;
```

Comment:
I don't think we need to mock caseData, since it's not a service. it's a data object, so we can build it to contain the data we want for the test